<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigLoader</class>
 <widget class="QWidget" name="ConfigLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="blockFileLine"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="otherLoadButton">
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="otherFileLine"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="otherDescription">
     <property name="text">
      <string>Other Label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="blockDescription">
     <property name="text">
      <string>Block Label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="blockLoadButton">
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="handlerFileLine"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="handlerDescription">
     <property name="text">
      <string>Handler Label</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="handlerLoadButton">
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="blockLabel">
     <property name="text">
      <string>Block Config</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="handlerLabel">
     <property name="text">
      <string>Handler Config</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="otherLabel">
     <property name="text">
      <string>Other Config</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="windowDescription">
     <property name="text">
      <string>Select Files to be loaded.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="loadConfigsButton">
     <property name="text">
      <string>Load Configs</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
